diff --git a/node_modules/buffers/index.js b/node_modules/buffers/index.js
index 86a9696..e185a7a 100644
--- a/node_modules/buffers/index.js
+++ b/node_modules/buffers/index.js
@@ -75,13 +75,13 @@ Buffers.prototype.splice = function (i, howMany) {
             removed.push(buffers[ii].slice(start, start + howMany));
             
             var orig = buffers[ii];
-            //var buf = new Buffer(orig.length - howMany);
-            var buf0 = new Buffer(start);
+            //var buf = Buffer.alloc(orig.length - howMany);
+            var buf0 = Buffer.alloc(start);
             for (var i = 0; i < start; i++) {
                 buf0[i] = orig[i];
             }
             
-            var buf1 = new Buffer(orig.length - start - howMany);
+            var buf1 = Buffer.alloc(orig.length - start - howMany);
             for (var i = start + howMany; i < orig.length; i++) {
                 buf1[ i - howMany - start ] = orig[i]
             }
@@ -146,7 +146,7 @@ Buffers.prototype.slice = function (i, j) {
         si ++
     ) { startBytes += buffers[si].length }
     
-    var target = new Buffer(j - i);
+    var target = Buffer.alloc(j - i);
     
     var ti = 0;
     for (var ii = si; ti < j - i && ii < buffers.length; ii++) {
@@ -193,7 +193,7 @@ Buffers.prototype.set = function set (i, b) {
 
 Buffers.prototype.indexOf = function (needle, offset) {
     if ("string" === typeof needle) {
-        needle = new Buffer(needle);
+        needle = Buffer.from(needle);
     } else if (needle instanceof Buffer) {
         // already a buffer
     } else {
