diff --git a/node_modules/unzip-stream/lib/matcher-stream.js b/node_modules/unzip-stream/lib/matcher-stream.js
index 6818160..548e4f4 100644
--- a/node_modules/unzip-stream/lib/matcher-stream.js
+++ b/node_modules/unzip-stream/lib/matcher-stream.js
@@ -14,7 +14,7 @@ function MatcherStream(patternDesc, matchFn) {
     this.requiredLength = this.pattern.length;
     if (patternDesc.requiredExtraSize) this.requiredLength += patternDesc.requiredExtraSize;
 
-    this.data = new Buffer('');
+    this.data = Buffer.alloc(0);
     this.bytesSoFar = 0;
 
     this.matchFn = matchFn;
@@ -57,7 +57,7 @@ MatcherStream.prototype.checkDataChunk = function (ignoreMatchZero) {
 
     var finished = this.matchFn ? this.matchFn(this.data, this.bytesSoFar) : true;
     if (finished) {
-        this.data = new Buffer('');
+        this.data = Buffer.alloc(0);
         return;
     }
 
diff --git a/node_modules/unzip-stream/lib/unzip-stream.js b/node_modules/unzip-stream/lib/unzip-stream.js
index 1a503b5..19cfe30 100644
--- a/node_modules/unzip-stream/lib/unzip-stream.js
+++ b/node_modules/unzip-stream/lib/unzip-stream.js
@@ -44,7 +44,7 @@ function UnzipStream(options) {
     stream.Transform.call(this);
 
     this.options = options || {};
-    this.data = new Buffer('');
+    this.data = Buffer.alloc(0);
     this.state = states.STREAM_START;
     this.skippedBytes = 0;
     this.parsedEntity = null;
@@ -306,7 +306,7 @@ UnzipStream.prototype._prepareOutStream = function (vars, entry) {
     };
 
     if (!fileSizeKnown) {
-        var pattern = new Buffer(4);
+        var pattern = Buffer.alloc(4);
         pattern.writeUInt32LE(SIG_DATA_DESCRIPTOR, 0);
         var zip64Mode = vars.extra.zip64Mode;
         var extraSize = zip64Mode ? 20 : 12;
@@ -662,7 +662,7 @@ UnzipStream.prototype._parseOrOutput = function (encoding, cb) {
                 this.data = this.data.slice(remaining);
             } else {
                 packet = this.data;
-                this.data = new Buffer('');
+                this.data = Buffer.alloc(0);
             }
 
             this.outStreamInfo.written += packet.length;
@@ -675,7 +675,7 @@ UnzipStream.prototype._parseOrOutput = function (encoding, cb) {
             }
         } else {
             var packet = this.data;
-            this.data = new Buffer('');
+            this.data = Buffer.alloc(0);
 
             this.outStreamInfo.written += packet.length;
             var outputStream = this.outStreamInfo.stream;
